src/app/shared/components/duration-input/duration-input.component.ts

```diff
@@ -10,6 +16,11 @@ import { ControlContainer, NG_VALUE_ACCESSOR } from '@angular/forms';
       multi: true,
       useExisting: forwardRef(() => DurationInputComponent),
     },
+    {
+      provide: NG_VALIDATORS,
+      multi: true,
+      useExisting: forwardRef(() => DurationInputComponent),
+    },
   ],
 })
 export class DurationInputComponent implements OnInit {
```

Comment:
See my comment about the implementation of ControlValueAccessor. Here is the same mistake.

Reply:
fixed